starkinfra/issuingtoken/__issuingtoken.py

```diff
@@ -0,0 +1,238 @@
+    ## Return:
+    - canceled IssuingToken object
+    """
+    return rest.delete_id(resource=_resource, id=id, user=user)
+
+
+def parse(content, signature, user=None):
+
+    """# Create a single verified IssuingToken request from a content string
+    Use this method to parse and verify the authenticity of the request received at the informed endpoint.
+    Token requests are posted to your registered endpoint whenever IssuingTokens are received.
+    They present IssuingToken data that must be analyzed and answered with approval or declination.
+    If the provided digital signature does not check out with the StarkInfra public key, a stark.exception.InvalidSignatureException will be raised.
+    If the request is not answered within 2 seconds or is not answered with an HTTP status code 200 the IssuingToken will go through the pre-configured stand-in validation.
+    ## Parameters (required):
+    - content [string]: response content from request received at user endpoint (not parsed)
+    - signature [string]: base-64 digital signature received at response header "Digital-Signature"
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - Parsed IssuingToken object
+    """
+    return parse_and_verify(
+        content=content,
+        signature=signature,
+        user=user,
+        resource=_resource,
+        key="",
+    )
+
+
+def response_authorization(status, reason=None, activation_methods=None, design_id=None, tags=None):
+    """# Helps you respond IssuingToken authorization requests
+    ## Parameters (required):
+    - status [string]: sub-issuer response to the authorization. ex: "approved" or "denied"
+    ## Parameters (conditionally required):
+    - reason [string, default ""]: denial reason. Options: "other", "bruteForce", "subIssuerError", "lostCard", "invalidCard", "invalidHolder", "expiredCard", "canceledCard", "blockedCard", "invalidExpiration", "invalidSecurityCode", "missingTokenAuthorizationUrl", "maxCardTriesExceeded", "maxWalletInstanceTriesExceeded"
+    - activation_methods [list of dictionaries, default None]: list of dictionaries with "type":string and "value":string pairs
+    - design_id [string, default None]: design unique id. ex: "5656565656565656"
+    ## Parameters (optional):
+    - tags [list of strings, default None]: tags to filter retrieved object. ex: ["tony", "stark"]
+    ## Return:
+    - Dumped JSON string that must be returned to us on the IssuingToken request
+    """
+    params = {"authorization": {
+        "status": status,
+        "reason": reason or "",
+        "activationMethods": activation_methods,
+        "designId": design_id,
+        "tags": tags,
+    }}
+    return dumps(api_json(params))
+
+
+def response_activation(status, reason=None, tags=None):
+    """# Helps you respond IssuingToken activation requests
```

Comment:
When a new token activation is triggered by your user, a POST request will be made to your registered URL for you to confirm the activation code you informed to them. You may identify this request through the present activation_code in the payload.
The POST request must be answered in the following format, within 2 seconds, and with an HTTP status code 200.